tests/draft2019-09/core/id.json

```diff
@@ -0,0 +1,138 @@
+[
+    {
+        "description": "Invalid use of fragments in location-independent $id",
+        "schema": {"$ref": "https://json-schema.org/draft/2019-09/schema"},
+        "tests": [
+            {
+                "description": "Identifier name",
+                "data": {
+                    "$ref": "#foo",
+                    "$defs": {
+                        "A": {
+                            "$id": "#foo",
+                            "type": "integer"
+                        }
+                    }
+                },
+                "valid": false
+            },
+            {
+                "description": "Identifier path",
+                "data": {
+                    "$ref": "#/a/b",
+                    "$defs": {
+                        "A": {
+                            "$id": "#/a/b",
+                            "type": "integer"
+                        }
+                    }
+                },
+                "valid": false
+            },
+            {
+                "description": "Identifier name with absolute URI",
```

Comment:
Isn't the point here (on the description) that it contains a non-empty fragment? Not that it's an absolute URI?

Reply:
I’m equating “identifier name” with “plain name”, a fragment. I could change to “plain name” if that’s more clear.

Reply:
Actually, looking at the test group description: "Invalid use of fragments in location-independent $id" tells me that this whole group is related to fragments. However, I'll change all the "Identifier name" to "Identifier plain name".